File: SDKs/Aspose.Words-Cloud-SDK-for-Java/src/main/java/com/aspose/words/model/ProtectionRequest.java
package com.aspose.words.model;

public class ProtectionRequest {
  private String Password = null;
  private String NewPassword = null;
  private String ProtectionType = null;
  /**
	 * getPassword
	 * Gets String
	 * @return Password
	 */
  public String getPassword() {
    return Password;
  }

	/**
	 * setPassword
	 * Sets String
	 * @param Password String
	 */
  public void setPassword(String Password) {
    this.Password = Password;
  }

  /**
	 * getNewPassword
	 * Gets String
	 * @return NewPassword
	 */
  public String getNewPassword() {
    return NewPassword;
  }

	/**
	 * setNewPassword
	 * Sets String
	 * @param NewPassword String
	 */
  public void setNewPassword(String NewPassword) {
    this.NewPassword = NewPassword;
  }

  /**
	 * getProtectionType
	 * Gets String
	 * @return ProtectionType
	 */
  public String getProtectionType() {
    return ProtectionType;
  }

	/**
	 * setProtectionType
	 * Sets String
	 * @param ProtectionType String
	 */
  public void setProtectionType(String ProtectionType) {
    this.ProtectionType = ProtectionType;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ProtectionRequest {\n");
    sb.append("  Password: ").append(Password).append("\n");
    sb.append("  NewPassword: ").append(NewPassword).append("\n");
    sb.append("  ProtectionType: ").append(ProtectionType).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words-Cloud-SDK-for-.NET/src/Com/Aspose/Words/Model/ProtectionRequest.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Com.Aspose.Words.Model 
{
  /// <summary>
  /// Request on changing of protection
  /// </summary>  
  public class ProtectionRequest 
  {                       
        /// <summary>
        /// Current password
        /// </summary>
        /// <value>Current password</value>    
        public string Password { get; set; }

        /// <summary>
        /// New password
        /// </summary>
        /// <value>New password</value>    
        public string NewPassword { get; set; }

        /// <summary>
        /// New type of protection
        /// </summary>
        /// <value>New type of protection</value>    
        public string ProtectionType { get; set; }

        /// <summary>
        /// Get the string presentation of the object
        /// </summary>
        /// <returns>String presentation of the object</returns>
        public override string ToString()  
        {
          var sb = new StringBuilder();
          sb.Append("class ProtectionRequest {\n");
          sb.Append("  Password: ").Append(Password).Append("\n");
          sb.Append("  NewPassword: ").Append(NewPassword).Append("\n");
          sb.Append("  ProtectionType: ").Append(ProtectionType).Append("\n");
          sb.Append("}\n");
          return sb.ToString();
        }
    }
}
